src/backend/empty/src/lib.rs

```diff
@@ -575,7 +575,7 @@ impl device::Device<Backend> for Device {
 pub struct QueueFamily;
 impl queue::QueueFamily for QueueFamily {
     fn queue_type(&self) -> queue::QueueType {
```

Comment:
this type and function should no longer be called "queue type". It's a set of flags now, so `QueueFlags`?

Reply:
name changed